Sources/plutil/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+add_executable(plutil
+	main.swift)
+
+target_link_libraries(plutil PRIVATE
+	Foundation)
+
+target_link_options(plutil PRIVATE
+    "SHELL:-no-toolchain-stdlib-rpath")
+
+set_target_properties(plutil PROPERTIES
+	INSTALL_RPATH "$ORIGIN/../lib/swift/${SWIFT_SYSTEM_NAME}")
+
+set_property(GLOBAL APPEND PROPERTY Foundation_EXPORTS plutil)
+install(TARGETS plutil
+	DESTINATION ${CMAKE_INSTALL_BINDIR})
```

Comment:
(nit) missing new line

Reply:
Fixed in 97442722cc9aa16a7d6961a86fc2d6e26a3493ad